EMacco/Customer-Ticketing-System
# #168825024 Setup front end

#### What does this PR do?
This PR sets up the environment for us to write and run the front end code

#### Description of Task to be completed?
- Setup react
- Setup redux
- Setup react router

#### How should this be manually tested?
- Checkout to this branch
- Start the servers `foreman start`
- Go to `http://localhost:3000` on your browser
- You should see `This is the home page`

#### What are the relevant pivotal tracker stories?
[#168825024](https://www.pivotaltracker.com/story/show/168825024)
